packages/ses/src/lockdown-shim.js

```diff
@@ -191,26 +191,23 @@ export const repairIntrinsics = (options = {}) => {
     );
   }
 
-  assert(
-    evalTaming === 'unsafeEval' ||
-      evalTaming === 'safeEval' ||
-      evalTaming === 'noEval',
-    d`lockdown(): non supported option evalTaming: ${q(evalTaming)}`,
-  );
+  evalTaming === 'unsafeEval' ||
+    evalTaming === 'safeEval' ||
+    evalTaming === 'noEval' ||
+    Fail`lockdown(): non supported option evalTaming: ${q(evalTaming)}`;
 
   // Assert that only supported options were passed.
   // Use Reflect.ownKeys to reject symbol-named properties as well.
   const extraOptionsNames = ownKeys(extraOptions);
-  assert(
-    extraOptionsNames.length === 0,
-    d`lockdown(): non supported option ${q(extraOptionsNames)}`,
-  );
+  extraOptionsNames.length === 0 ||
+    Fail`lockdown(): non supported option ${q(extraOptionsNames)}`;
 
-  assert(
-    priorLockdown === undefined,
-    `Already locked down at ${priorLockdown} (SES_ALREADY_LOCKED_DOWN)`,
-    TypeError,
-  );
+  priorLockdown === undefined ||
+    // eslint-disable-next-line @endo/no-polymorphic-call
+    assert.fail(
+      d`Already locked down at ${priorLockdown} (SES_ALREADY_LOCKED_DOWN)`,
+      TypeError,
```

Comment:
Reviewers,
I left this as `assert.fail(d...)` rather than converting to `Fail...` because of the additional `TypeError`. This occurs several places. For each of these, if the `TypeError` is extraneous and you would prefer that I convert to the `Fail...` form, please let me know.